about/contact/index.html

```diff
@@ -40,50 +40,50 @@ <h4>AGRC Staff Information</h4>
       <td colspan="4" style="padding:0;height:1px;border:0;background-color:#e67e15"></td>
     </tr>
     <tr>
-      <th class="text-left" colspan="4"><strong>Director</strong></th>
```

Comment:
Shouldn't this continue to say director?

Reply:
I think that "AGRC Team Lead" @brigsz's preferred title over "Director".

Reply:
take me to your leader

Reply:
👽